concat.js

```diff
@@ -125,18 +140,42 @@ Concat.prototype.build = function() {
       beginSection();
       concat.addSpace(this.footer + '\n');
     }
+
+    this._lastTree = this.getFSTree();
```

Comment:
we should walk twice if we do a build, this seems not ideal. Isn't the new `this._lastTree` the one fetched in `shouldBuild` via `this.getFSTree()`?